narwhal/types/src/primary.rs

```diff
@@ -729,6 +727,12 @@ pub struct WorkerSynchronizeMessage {
     pub target: PublicKey,
 }
 
+/// Used by the primary to request that the worker delete the specified batches.
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub struct WorkerDeleteBatchesMessage {
```

Comment:
Should we name it `WorkerDeleteBatchesRequest` so that we distinguish between messages and RPCs?

E.g. we can adopt convention like RPCs taking XxxRequest and return XxxResponse (or just `()` if no response), and the XxxMessage is reserved for messages that are sent via `[un]reliable_send`

Reply:
The convention I adopted (which is also used by Synchronize) is that `*Message` is for RPCs with an empty response and `*{Request,Response}` is for RPCs with a response. Hoping to continue migrating things to this convention. I think whether it's sent reliably or not shouldn't have to do with the message type names, since the type names can't enforce anything about how it's sent.